pmd-cpp/src/test/resources/net/sourceforge/pmd/cpd/cpp/issue-1559.cpp

```diff
@@ -4,7 +4,7 @@ namespace ABC
   {
 
 #ifdef USE_QT
-    const char* perPixelQml = R"QML(
+    const char* perPixelQml = "QML(
```

Comment:
I see, with your fix, this is not a parsing error anymore... Can you add there a small comment saying sth. like "// provoking a parser error"?